site2/docs/client-libraries-cpp.md

```diff
@@ -104,6 +139,13 @@ If you get an error like "libpulsar.so: cannot open shared object file: No such
 
 :::
 
+### APK
+
+```bash
+apk add --allow-untrusted ./apache-pulsar-client-3.0.0-r0.apk
```

Comment:
Also, it seems to me that `-r0` referred to a release candidate, is it a formal release?

Reply:
Yes. See https://archive.apache.org/dist/pulsar/pulsar-client-cpp-3.0.0/apk-x86_64/x86_64/